common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -29,11 +29,13 @@
     </li>
 }
 
-<div class="main-menu-container" id="main-menu">
+<label for="main-menu-toggle" class="main-menu-container__overlay" aria-hidden="true"></label>
```

Comment:
k remembered the form element is the tabbabable element so if you're avoiding tabbing to the checkbox this is cool innit.